### ISSUE TITLE
Converting Katalon/Selenium IDE recordings into Golem test scripts

### ISSUE THREAD

**Issue by ATTO-DT** [golemhq/golem#216]
This Feature should be implement like below.
1. user should have option on test code script page  for convert Katalon/Selenium IDE recording script into Golem test script
     need to add one button on **test code script page** where user can upload katon/selenium IDE script and after upload it should convert into golem test script and will display on test code script pge.

through command it should be like
 golem convert katalonscript.py <project_name> <test_name>

**Comment by ATTO-DT** [comment]
@luciano-renzi  what you thinking to implement this one I will go ahead to implement this feature.

**Comment by luciano-renzi** [comment]
I would give priority to importing from Selenium IDE rather than the other one. I think that would be a far more helpful feature. But it's up to you.

I see the Selenium IDE can export a single test as a pytest file or an entire project as a .side file (which looks like json). Which of these were you trying to use?

**Comment by Sunil-Rathore** [comment]
first we can start work on single test as a pytest file because i m trying to give  import option on test page so only pytest file we can import there.
for import .side need to add  import option on project page  that we can think about later.

**Comment by luciano-renzi** [comment]
In that case it should be better if the importer creates each test in the side file if a test with the same name does not yet exist in Golem otherwise it should ignore it

**Comment by Sunil-Rathore** [comment]
@luciano-renzi  yes this feature almost completed but what u think about if project name already exists. 
actually side will import with first  new project name and after test and suite inside created project.

there is below three different way to handle this one.
if project already exist then it will not import again.
if project already exist and user want to add newly test and suite in the existing project.
if project already exist and user want to replace the already exist test or suite in existing project.

let me know your comments which one is good as of now.

**Comment by luciano-renzi** [comment]
if project does not exist, a new project should be created.
for each test if a test with same name exist, show warning. it could have a --overwrite flag to replace existing test with new one.
